client/component/comment.js

```diff
@@ -213,14 +215,23 @@ function EditComment({comment, big=false, setComment, topLevel, onEditingDone, o
             onEditingDone(comment);
         }
     }
+    const {isMobile} = getDeviceInfo();
+    const {isTextFieldFocused} = useFocus();
```

Comment:
Probably better to get the focus state from a onFocusChance callback on TextField. Contexts are super-useful sometimes, but feel like overkill here.

Reply:
done